app/styles/app.scss

```diff
@@ -340,6 +340,26 @@ ul.preprints-block-list {
     padding-left: 0;
 }
 
+#sortBy {
+    min-width: 291px;
+    width: 100%;
+}
+
+#sortByOptionList {
+    background-color: #EFEFEF;
```

Comment:
I'm not sure if this is the only place where we have a dropdown on preprints, but it's worth looking into for consistency. In #285 I added one for selecting preprint file versions, so it should be consistent. I'd prefer to modify the global dropdown styling, instead of having one-off styling just for this particular dropdown.